06_puppy/alfredxiao/syncstore.go

```diff
@@ -0,0 +1,51 @@
+package main
+
+import (
+	"fmt"
+	"sync"
+)
+
+type syncStore struct {
```

Comment:
Interesting choice to make this private.

Reply:
Don't make it private :)